projects/storefrontlib/src/shared/components/replenishment-order-cancellation-dialog/replenishment-order-cancellation-dialog.component.ts

```diff
@@ -89,5 +97,6 @@ export class ReplenishmentOrderCancellationDialogComponent
 
   ngOnDestroy(): void {
     this.subscription.unsubscribe();
+    this.userReplenishmentOrderService.clearCancelReplenishmentOrderProcessState();
```

Comment:
This fires each time you close the dialog even if you click back on the dialog. Is there a reason for this?

Reply:
At first, I put it in the success, but I had the idea of maybe if the dialog closes for any reason, than we just fire that action at any point.

Reply:
That's also why I assumed it was the case. Not much harm in being extra careful.